src/app/containers/MediaPlayer/wrapper/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
```

Comment:
Hopefully by separating this into its own component, there is no business logic directly tied to it, so it can just be lifted and shifted into Psammead.